### PR TITLE
Move leftout functions from API/common/utils.py

### PR DESCRIPTION
**Pull request by Achie72**
Some functions were left in API/common/utils.py, moved them to their dedicated spot under API/builder/utils.py and API/testrunner/utils.py

JSRemoteTest-DCO-1.0-Signed-off-by: Bela Toth tbela@inf.u-szeged.hu

### CODE REVIEW

File: jstest/builder/utils.py [hunk 1]
```diff
@@ -43,9 +58,91 @@ def create_build_info(env):
     # Merge the collected values into a result object.
     build_info = {
         'build-date': utils.current_date('%Y-%m-%dT%H.%M.%SZ'),
+        'build-date': get_standardized_date(),
```
Comment (robertsipka) [inline jstest/builder/utils.py]: You will overwrite the first `build-date` above. But I think this patch was not properly rebased to the master. The `get_standardized_date` function was removed in #177.

File: jstest/builder/targets/stm32f4dis.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 from jstest.builder import builder
 from jstest.common import utils
-
+from jstest.builder import utils as builderUtils
```
Comment (rtakacs) [inline jstest/builder/targets/stm32f4dis.py]: Please use snake_case style.

File: jstest/testrunner/devices/artik053.py [hunk 3]
```diff
@@ -18,6 +18,7 @@
 from jstest.testrunner.devices.device_base import RemoteDevice
 from jstest.common import utils
 from jstest.testrunner.devices.connections.serialcom import SerialConnection
+from jstest.testrunner import utils as testrunnerUtils
```
Comment (rtakacs) [inline jstest/testrunner/devices/artik053.py]: +1 for snake_case style for testrunnerUtils.

File: jstest/testrunner/devices/artik530.py [hunk 4]
```diff
@@ -17,6 +17,7 @@
 
 from jstest.testrunner.devices.device_base import RemoteDevice
 from jstest.common import utils, paths
+from jstest.testrunner import utils as testrunnerUtils
```
Comment (rtakacs) [inline jstest/testrunner/devices/artik530.py]: Ditto.

File: jstest/testrunner/devices/device_base.py [hunk 5]
```diff
@@ -16,6 +16,7 @@
 import time
 
 from jstest.common import console, utils
+from jstest.testrunner import utils as testrunnerUtils
```
Comment (rtakacs) [inline jstest/testrunner/devices/device_base.py]: Ditto.

File: jstest/builder/utils.py [hunk 6]
```diff
@@ -10,9 +10,24 @@
 # distributed under the License is distributed on an "AS IS" BASIS
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
-# limitations under the License.
```
Comment (rtakacs) [inline jstest/builder/utils.py]: You have removed the end of the license.

File: jstest/builder/utils.py [hunk 7]
```diff
@@ -10,9 +10,24 @@
...
+
+import os
+import time
```
Comment (rtakacs) [inline jstest/builder/utils.py]: `time` module is not used.

**Comment by robertsipka** [comment]
@Achie72  Please rebase it with master.

**Pull request merged**